metaverse_components/use.js

```diff
@@ -0,0 +1,45 @@
+import * as THREE from 'three';
```

Comment:
Why break out sound only into the component? There is a lot of `use` behavior we can move to the `use` component, but this simply makes the code worse for now since it fragments the behavior even more.

If we are doing to break out `use` into a component, we should do it for all use behavior, not just sounds.